account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/UpdateEmailHandler.java

```diff
@@ -17,12 +17,8 @@
 import uk.gov.di.accountmanagement.services.CodeStorageService;
 import uk.gov.di.authentication.shared.entity.ErrorResponse;
 import uk.gov.di.authentication.shared.exceptions.UserNotFoundException;
-import uk.gov.di.authentication.shared.helpers.ClientSubjectHelper;
-import uk.gov.di.authentication.shared.helpers.IpAddressHelper;
+import uk.gov.di.authentication.shared.helpers.*;
```

Comment:
We should [avoid wildcard imports](https://gds.slack.com/archives/C01ELVA4K1U/p1713448065518759?thread_ts=1713434626.132039&cid=C01ELVA4K1U)